src/components/pages/PointsDashboard/PointsDashboard.tsx

```diff
@@ -365,16 +365,21 @@ const QuestList = () => {
         questType,
         questChainId,
         questCurrencies,
+        startAt,
     }: {
         questType: QuestType;
         questChainId?: number | null;
         questCurrencies?: string[] | null;
+        startAt: string | null;
     }) => {
         switch (questType) {
             case QuestType.Deposit:
                 return (
                     <Button
                         size={ButtonSizes.md}
+                        disabled={
+                            startAt ? dayjs().isBefore(dayjs(startAt)) : false
```

Comment:
if the startAt is null, the button is enabled. Is it intended?

Reply:
It is intended.